### ISSUE TITLE
low accuracy with spatial pretrained model

### ISSUE THREAD

**Issue by hoangdzung** [jeffreyhuang1/two-stream-action-recognition#29]
Hi! I'm training your spatial_cnn model. I downloaded rgb images from "feichtenhofer/twostreamfusion". Then, I followed your guide (modify some functions) and used your pretrained model, but I got only 79.xx% on test set and 99.xx% on training set. I can't achieve your result, which is 82.1%. 
So what's my problem? Thank you.

**Comment by hoangdzung** [comment]
I evaluated spatial cnn model with starting epoch is 31 and I got your result. If I continue training the model, accuracy goes down. 
At least I can achieve your result. Thank you.

**Comment by duygusar** [comment]
When I train the network, the validation loss oscillates like crazy and never really improves? What is the measurement of accuracy here?It seems like the model never converges even when I start training with the pretrained model and resume. Also, as mentioned here, I can not only do the testing - https://github.com/jeffreyhuang1/two-stream-action-recognition/issues/41
